src/Sentry.Serilog/SentrySinkExtensions.cs

```diff
@@ -159,132 +205,45 @@ public static class SentrySinkExtensions
-        {
-            sentrySerilogOptions.MinimumEventLevel = minimumEventLevel.Value;
-        }
-
-        if (minimumBreadcrumbLevel.HasValue)
-        {
-            sentrySerilogOptions.MinimumBreadcrumbLevel = minimumBreadcrumbLevel.Value;
-        }
-
-        if (formatProvider != null)
-        {
-            sentrySerilogOptions.FormatProvider = formatProvider;
-        }
-
-        if (textFormatter != null)
-        {
-            sentrySerilogOptions.TextFormatter = textFormatter;
-        }
-
-        if (sendDefaultPii.HasValue)
-        {
-            sentrySerilogOptions.SendDefaultPii = sendDefaultPii.Value;
-        }
-
-        if (isEnvironmentUser.HasValue)
-        {
-            sentrySerilogOptions.IsEnvironmentUser = isEnvironmentUser.Value;
-        }
-
-        if (!string.IsNullOrWhiteSpace(serverName))
-        {
-            sentrySerilogOptions.ServerName = serverName;
-        }
-
-        if (attachStackTrace.HasValue)
-        {
-            sentrySerilogOptions.AttachStacktrace = attachStackTrace.Value;
-        }
-
-        if (maxBreadcrumbs.HasValue)
-        {
-            sentrySerilogOptions.MaxBreadcrumbs = maxBreadcrumbs.Value;
-        }
-
-        if (sampleRate.HasValue)
-        {
-            sentrySerilogOptions.SampleRate = sampleRate;
-        }
-
-        if (!string.IsNullOrWhiteSpace(release))
-        {
-            sentrySerilogOptions.Release = release;
-        }
-
-        if (!string.IsNullOrWhiteSpace(environment))
-        {
-            sentrySerilogOptions.Environment = environment;
-        }
-
-        if (maxQueueItems.HasValue)
-        {
-            sentrySerilogOptions.MaxQueueItems = maxQueueItems.Value;
-        }
-
-        if (shutdownTimeout.HasValue)
-        {
-            sentrySerilogOptions.ShutdownTimeout = shutdownTimeout.Value;
-        }
-
-        if (decompressionMethods.HasValue)
-        {
-            sentrySerilogOptions.DecompressionMethods = decompressionMethods.Value;
-        }
-
-        if (requestBodyCompressionLevel.HasValue)
-        {
-            sentrySerilogOptions.RequestBodyCompressionLevel = requestBodyCompressionLevel.Value;
-        }
-
-        if (requestBodyCompressionBuffered.HasValue)
-        {
-            sentrySerilogOptions.RequestBodyCompressionBuffered = requestBodyCompressionBuffered.Value;
-        }
-
-        if (debug.HasValue)
-        {
-            sentrySerilogOptions.Debug = debug.Value;
-        }
-
-        if (diagnosticLevel.HasValue)
-        {
-            sentrySerilogOptions.DiagnosticLevel = diagnosticLevel.Value;
-        }
-
-        if (reportAssembliesMode.HasValue)
-        {
-            sentrySerilogOptions.ReportAssembliesMode = reportAssembliesMode.Value;
-        }
-
-        if (deduplicateMode.HasValue)
-        {
-            sentrySerilogOptions.DeduplicateMode = deduplicateMode.Value;
-        }
+        sentrySerilogOptions.Dsn = dsn ?? sentrySerilogOptions.Dsn;
```

Comment:
Is this just to be "in line" and look more concise or am I missing something with overwriting?

Reply:
It's trying to "only set `dsn` as a value if isn't `null` otherwise keep `option.Dsn` as-is.
So looks like the original code, just refactored.

I personally don't prefer this, before it was pretty clear (if not null, set). Now it's set! Either this value or the other. Properties could have side effects, etc. Just doing some work so it can be 1 line instead of 3 per property`

Reply:
Yeah logically it should do exactly the same thing as the old code (assuming the setters don't have any side effects)... but it does result in an unecessary `sentrySerilogOptions.Dsn = sentrySerilogOptions.Dsn` when `sentrySerilogOptions.Dsn is not null`. 

I can revert this if you guys want. I liked it because it 1/3 the amount of code (visually) and I just generally like stuff concise.